sqldelight-studio-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/lang/SqlDelightFileViewProviderFactory.kt

```diff
@@ -98,7 +98,6 @@ internal class SqlDelightFileViewProvider(virtualFile: VirtualFile, language: La
 
   companion object {
     private val localFileSystem = LocalFileSystem.getInstance()
-    private val sqliteCompiler = SqliteCompiler()
     private val sqldelightValidator = SqlDelightValidator()
```

Comment:
Looks like we can probably give this the static method / instance state treatment next?